client/coral-admin/src/components/KarmaTooltip.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { Icon } from 'coral-ui';
+import styles from './KarmaTooltip.css';
+import ClickOutside from 'coral-framework/components/ClickOutside';
+import t from 'coral-framework/services/i18n';
+
+const initialState = { menuVisible: false };
+
+class KarmaTooltip extends React.Component {
+  static propTypes = {
+    settings: PropTypes.shape({
+      reliable: PropTypes.number.isRequired,
+      unreliable: PropTypes.number.isRequired,
+    }).isRequired,
+  };
+
+  state = initialState;
+
+  toogleMenu = () => {
+    this.setState({ menuVisible: !this.state.menuVisible });
+  };
+
+  hideMenu = () => {
+    this.setState({ menuVisible: false });
+  };
+
+  render() {
+    const { settings: { unreliable } } = this.props;
+    const { menuVisible } = this.state;
+
+    return (
+      <ClickOutside onClickOutside={this.hideMenu}>
+        <div className={cn(styles.karmaTooltip, 'talk-admin-karma-tooltip')}>
+          <span
+            onClick={this.toogleMenu}
+            className={cn(styles.icon, 'talk-admin-karma-tooltip-icon')}
+          >
+            <Icon name="info" />
+          </span>
+
+          {menuVisible && (
+            <div className={cn(styles.menu, 'talk-admin-karma-tooltip-menu')}>
+              <strong>{t('user_detail.user_karma_score')}</strong>
+              <ul>
+                {/* <li>
```

Comment:
Reason this is commented out?

Reply:
For now, we wanted to hide the labels for reliable + neutral because we don't do anything with it. When we do, it may make sense to display them. Commenting them out here leaves it there so we can restore it easily going forward.

Reply:
Ok, I was just thinking if we should add comments there whenever we comment parts of code out so it just don't look like it's forgotten :-P, e.g. "wyatt: kept here for future use"